json/pom.xml

```diff
@@ -73,6 +73,12 @@
             <version>${commons-collections4.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+			<groupId>org.assertj</groupId>

```

Comment:
Please check the whitespacing here. Looks like you are using tabs.

Reply:
I have used the XML file provided in the documentation as default eclipse format config. Though it seems, that added few extra whitespaces. Is it fine or shall I need to format this again?

Reply:
@kdsalvy, I recommend keeping the predominant format already in the file (just so the PR is focused). Since the pom originally was using spaces, let's keep using spaces for it. We can standardize the formatting at another time.

Reply:
@jzheaux, I have made the changes as expected by you.